components/header.js

```diff
@@ -6,13 +6,15 @@ export default function Header({ subtitle }) {
     <div className="flex-row flex items-left text-left">
       <Link href="/">
         <a className="flex title-font font-medium items-center md:justify-start justify-center mr-4">
-          <svg xmlns="http://www.w3.org/2000/svg" fill="none" stroke="currentColor" strokeLinecap="round" strokeLinejoin="round" strokeWidth="2" className="w-10 h-10 text-white p-2 bg-red-500 hover:bg-red-600 rounded-full" viewBox="0 0 24 24">
-            <path strokeLinecap="round" strokeLinejoin="round" strokeWidth="2" d="M3 12l2-2m0 0l7-7 7 7M5 10v10a1 1 0 001 1h3m10-11l2 2m-2-2v10a1 1 0 01-1 1h-3m-6 0a1 1 0 001-1v-4a1 1 0 011-1h2a1 1 0 011 1v4a1 1 0 001 1m-6 0h6" />
-          </svg>
+          <img src="/logo.png" className="w-14 h-14 p-2" />
         </a>
       </Link>
-      <a href="/articles" className="text-3xl text-red-500 hover:text-red-600 font-black hover:underline mr-4">Articles</a>
-      <a href="https://github.com/utensils" target="_blank" className="text-3xl text-red-500 hover:text-red-600 font-black hover:underline">GitHub</a>
+      <span className="inline-block align-middle">
+        <a href="/articles" className="text-3xl text-red-500 hover:text-red-600 font-black hover:underline mr-4">Articles</a>
+      </span>
+      <span className="inline-block align-middle">
+        <a href="https://github.com/utensils" target="_blank" className="text-3xl text-red-500 hover:text-red-600 font-black hover:underline">GitHub</a>
+      </span>
```

Comment:
@burden / @johnTurknett either of you want to take a peek at how to vertically align these on the logo? I can't seem to get both to do it 🤔

Reply:
![image](https://user-images.githubusercontent.com/73386/106846213-b63d4500-6669-11eb-8708-adbd3d4c6f44.png)

This was the best I got 😰

Reply:
![image](https://user-images.githubusercontent.com/73386/106846452-20ee8080-666a-11eb-8c1e-bc40c7914f6a.png)

Got it! 🚀